### ISSUE TITLE
HTTP code: 403

### ISSUE THREAD

**Issue by krishnazure** [kitematic/kitematic#847]
The hello-world-nginx works fine.. but Unable to create any other Docker images via Kitematic.
![failure to create docker image](https://cloud.githubusercontent.com/assets/5904129/8833846/e53ba5f8-307e-11e5-8f44-91cac7b75d43.jpg)

**Comment by FrenchBen** [comment]
Is this on Windows?

**Comment by krishnazure** [comment]
Yes , on Windows 8.1 Enterprise.

**Comment by krishnazure** [comment]
Interesting.. the issue got resolved automatically.. it appeared the container was getting downloaded in the background...  The containers started working after 30 odd minutes of wait... 

![error resolved](https://cloud.githubusercontent.com/assets/5904129/8838825/cbd2f976-309d-11e5-9dab-1a922aa4fef6.jpg)

**Comment by FrenchBen** [comment]
Some of the 403 generated errors are triggered when you are logged in to your Docker Hub account, but did not validate your account. 

I see from the screenshot that you were logged in, could it be that you validated your account after the fact, which allowed the images to be downloaded in the background?

**Comment by krishnazure** [comment]
I did validate the account , following are my observations. 
1. I tried with another image for Postgres. The error message still popups up. But the download is happening in the  background. 
2. The image download (rethinkDB) didn't get completed. It will be nice to have  status update on the actual download and how much is left.. 
3. The Http Code:403 might be changed to something more relevant.

![error message postgres](https://cloud.githubusercontent.com/assets/5904129/8839976/46228cae-30a9-11e5-91ea-e53e189d5e9f.jpg)
